<!-- -*- mode: XML -*- -->
<launch>
    <group ns="/dmitry">
    <rosparam file="$(find robots_config)/dmitry/config_face.yaml" command="load"/>
    <node name="dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" output="screen">
        <rosparam>
            namespace: dxl_manager
            serial_ports:
                pan_tilt_port:
                    port_name: "/dev/ttyUSB0"
                    baud_rate: 117647
                    min_motor_id: 0
                    max_motor_id: 5
                    update_rate: 20
        </rosparam>
     </node>
     <node name="tilt_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port pan_tilt_port
                rotate_controller"
          output="screen"/>
    <node name="pitch1_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port pan_tilt_port
                pitch1_controller"
          output="screen"/>
    <node name="roll_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port pan_tilt_port
                roll_controller"
          output="screen"/>
    <node name="pitch2_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port pan_tilt_port
                pitch2_controller"
          output="screen"/>
    <node name="jaw_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port pan_tilt_port
                jaw_controller"
          output="screen"/> -->
    <node name="pau2motors" pkg="pau2motors" type="pau2motors_node.py"
          output="screen"/>
    <node name="basic_head_api" pkg="basic_head_api" type="head_ctrl.py"
          output="screen"/>
    <node name="dmitry_eyes" pkg="ros_pololu_servo" type="ros_pololu_servo_node" output="screen">
          <param name="port" value="/dev/serial/by-id/usb-Pololu_Corporation_Pololu_Mini_Maestro_24-Channel_USB_Servo_Controller_00080340-if00" />
    </node>
    <node name="dmitry_face" pkg="ros_pololu_servo" type="ros_pololu_servo_node" output="screen">
          <param name="port" value="/dev/serial/by-id/usb-Pololu_Corporation_Pololu_Mini_Maestro_24-Channel_USB_Servo_Controller_00080333-if00" />
    </node>
    </group>
</launch>
